### PR TITLE
Return a 404 when there is no paging estimate found for the given…

### LINKED ISSUE #269: Set fall-back text for paging schedule when no schedule is available.

### PR DESCRIPTION
**Pull request by jkeck**
…locations.

Closes #269 

### Before
business-before

### After
business-after

### CODE REVIEW

File: app/controllers/paging_schedule_controller.rb [hunk 1]
```diff
@@ -6,10 +6,13 @@ class PagingScheduleController < ApplicationController
 
   def show
     schedule = PagingSchedule.for(request_for_schedule)
```
Comment (cbeer) [inline app/controllers/paging_schedule_controller.rb]: I'm happy with this, but is not having a paging schedule exceptional (and we could rescue the exception with a 404..)?

Comment (jkeck) [inline app/controllers/paging_schedule_controller.rb]: :+1: see below

File: app/controllers/paging_schedule_controller.rb [hunk 2]
```diff
@@ -6,11 +6,12 @@ class PagingScheduleController < ApplicationController
...
-    return unless schedule.present?
     respond_to do |format|
       format.json { render json: schedule.estimate }
       format.html { render text: schedule.estimate.to_s }
     end
+  rescue PagingSchedule::ScheduleNotFound
```
Comment (cbeer) [inline app/controllers/paging_schedule_controller.rb]: At the controller level using `rescue_from`?

Comment (jkeck) [inline app/controllers/paging_schedule_controller.rb]: :ok_hand:

**Comment by cbeer** [comment]
:+1:

**Pull request merged**